ramoun/chapter_02/ex2_5.cpp

```diff
@@ -0,0 +1,15 @@
+// ex2_5.cpp
+// check if lowercase
+#include <iostream>
+#include <ctype.h>
+
+using namespace std;
+
+int main() {
```

Comment:
### better be consistent with coding style

- braces on the same level as code level
```cpp
int main () 
    {
    // code level
    }
```
- braces on their own level:
```cpp
int main() {
    // code level
}
```